apps/nextra/pages/en/build/smart-contracts.mdx

```diff
@@ -68,7 +68,6 @@ Here is a `hello_blockchain` example of move
 ### Move IDE plugins
 
 - [Aptos Move Analyzer](https://marketplace.visualstudio.com/items?itemName=MoveBit.aptos-move-analyzer) for Visual Studio.
-- [Remix IDE Plugin](../community/contributions/remix-ide-plugin.md): {/* TODO: fix reference link */} Offers a web-based development environment. It is a no-setup tool with a graphical interface for developing Move modules.
```

Comment:
I don't think we should be removing this one.  The remix plugin should still be here / we should move the doc page, or refer to their docs.

Reply:
Hmm ok. Where would you put it? Advanced Guides?

Reply:
Let me see if I can talk to them, and see if they have a guide somewhere else for it.  Don't think it belongs directly on this site.

Reply:
kk sounds good. In the meantime, do you think it'll be alright if we remove it. We can add it back in later once we've resolved the status for it?